R/utilities.R

```diff
@@ -3015,6 +3020,116 @@ get_bams = function(sample,
   return(bam_details)
 }
 
+#' @title View a variant in IGV
+#'
+#' @description Load bam(s) and view the context around a mutation
+#'
+#' @details 
+#'
+#' @param this_mutation Specify the mutation of interest in MAF format.
+#' @param igv_port Specify the port IGV is listening on.
+#' @param socket 
+#' @param sort_by base, quality, sample or readGroup
+#'
+#' @return Path to file (.png).
+#'
+#' @import SRAdb
+#' @export
+#'
+#' @examples
```

Comment:
I think this example is a good candidate for an example that should not be triggered with `build_site`. Consider wrapping the examples in `\dontrun{}`.

Reply:
Good call. Fixed.